src/homepageExperience/components/steps/Finish.tsx

```diff
@@ -24,50 +25,81 @@ export const Finish = (props: OwnProps) => {
   useEffect(() => {
     event(`firstMile.${props.wizardEventName}.finished`)
   }, [])
+
+  const fireConfetti = () => {
+    const count = 200
+    const defaults = {
+      origin: {y: 0.7},
+    }
+
+    const fire = (particleRatio, options) => {
+      confetti(
+        Object.assign({}, defaults, options, {
+          particleCount: Math.floor(count * particleRatio),
+        })
+      )
+    }
```

Comment:
fyi, since we make use of the spread operator, this can be changed to:

```ts
confetti({
  ...defaults,
  ...options,
  particleCount: Math.floor(count *  particleRatio)
})
```

Reply:
✅